From 6daaf8a10f2678f1325546992aa27552f0407c1e Mon Sep 17 00:00:00 2001
From: Sandro <devel@penguinpee.nl>
Date: Sun, 8 Dec 2024 11:22:42 +0100
Subject: [PATCH 1002/1005] Change issue URL to Bugzilla

Since upstream does not whish to receive bug reports from non-Flatpak
users, direct them to Bugzilla instead.
---
 data/com.usebottles.bottles.metainfo.xml.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/com.usebottles.bottles.metainfo.xml.in.in b/data/com.usebottles.bottles.metainfo.xml.in.in
index c380c823..fd2261d9 100644
--- a/data/com.usebottles.bottles.metainfo.xml.in.in
+++ b/data/com.usebottles.bottles.metainfo.xml.in.in
@@ -70,7 +70,7 @@
   <translation type="gettext">@APP_ID@</translation>
   <content_rating type="oars-1.1"/>
   <url type="homepage">https://usebottles.com</url>
-  <url type="bugtracker">https://github.com/bottlesdevs/Bottles/issues</url>
+  <url type="bugtracker">https://bugz.fedoraproject.org/bottles</url>
   <url type="help">https://docs.usebottles.com</url>
   <url type="donation">https://usebottles.com/funding</url>
   <url type="translate">https://hosted.weblate.org/engage/bottles</url>
-- 
2.51.0

